torch/testing/_internal/common_methods_invocations.py

```diff
@@ -865,6 +907,24 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
            supports_tensor_out=False,
            test_inplace_grad=False,
            sample_inputs_func=sample_inputs_broadcast_to),
+    TriangularOpInfo('cholesky_inverse',
+                     op=torch.cholesky_inverse,
+                     dtypes=floating_and_complex_types(),
+                     test_inplace_grad=False,
+                     supports_tensor_out=True,
+                     decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
+                     skips=(
+                         # RuntimeError: cholesky_inverse does not support automatic differentiation for outputs with complex dtype.
+                         SkipInfo('TestGradients', 'test_fn_grad',
+                                  dtypes=[torch.complex64, torch.complex128]),
+                         SkipInfo('TestGradients', 'test_fn_gradgrad',
+                                  dtypes=[torch.complex64, torch.complex128]),
+                         SkipInfo('TestCommon', 'test_variant_consistency_jit',
+                                  dtypes=[torch.complex64, torch.complex128]),
+                         SkipInfo('TestCommon', 'test_variant_consistency_eager',
```

Comment:
Does this test need to be skipped? I thought it would check if backward failed in both variants and still pass the test.

Reply:
This test needs to be skipped because a RuntimeError is thrown if input with require_grad=True is passed to cholesky_inverse. I will update the backward rule later.

Reply:
I think you also need `test_complex_grad=False` here

Reply:
I somehow missed that flag. Yes, I need to use that.